src/pino_transport.ts

```diff
@@ -1,44 +1,47 @@
-import { LOGGER_LEVEL_SEVERITY } from "./logger"
+import { LOGGER_LEVEL_SEVERITY, LOGGER_FORMAT } from "./logger"
 import build from "pino-abstract-transport"
-import { Client } from "./client"
+import { Extension } from "./extension"
 
 type PinoTransportOptions = {
-  client: Client
-  group: string
+  group?: string
 }
 
-const appsignalPinoTransport = ({
-  client,
-  group = "app"
-}: PinoTransportOptions) => {
+type LogData = {
+  severity: number
+  message: string
+  attributes: Record<string, any>
+}
+
+const appsignalPinoTransport = ({ group = "app" }: PinoTransportOptions) => {
```

Comment:
```suggestion
const appsignalPinoTransport = ({ group = "application" }: PinoTransportOptions) => {
```
Is this the default group value and should it be "application" then?

Reply:
Yes, it's the default group value. I'm not sure what it should be -- [the logger itself](https://github.com/appsignal/appsignal-nodejs/blob/main/src/client.ts#L137-L147) doesn't really have a default value for the group. `application` is the default _source_. I can't find any other AppSignal loggers that have default group values.

Note that I'm preserving [existing behaviour](https://github.com/appsignal/appsignal-nodejs/pull/1151/files/11b789d924788abf97460727da18ab75c611b361#diff-4e22a816fc6ab0d9f33925bbcd1c8116a17341f0e0232931bf69f0bc281a543bL22) here -- the default value was already `"app"`, I just moved where it's being set.